src/App.jsx

```diff
@@ -1,32 +1,24 @@
-import React, { useEffect } from 'react';
+import React from 'react';
+import { Switch, Route } from 'react-router-dom';
 
-import { useDispatch } from 'react-redux';
-
-import RegionsContainer from './RegionsContainer';
-import CategoriesContainer from './CategoriesContainer';
-import RestaurantsContainer from './RestaurantsContainer';
-
-import {
-  loadInitialData,
-} from './actions';
-
-// 0. 지역, 분류 목록을 얻기
-// 1. 지역 선택 - Regions <- API (0)
-// 2. 분류 선택 - Categories - 한식, 중식, 일식, ... <- API (0)
-// 3. 식당 목록 - Restaurants <- API (with region, category) -> 1, 2 모두 완료된 경우
+import Header from './Header';
+import Home from './Home';
+import About from './About';
+import Restaurants from './Restaurants';
+import RestaurantContainer from './RestaurantContainer';
+import NotFoundPage from './NotFoundPage';
 
 export default function App() {
-  const dispatch = useDispatch();
-
-  useEffect(() => {
-    dispatch(loadInitialData());
-  });
-
   return (
-    <div>
-      <RegionsContainer />
-      <CategoriesContainer />
-      <RestaurantsContainer />
-    </div>
+    <>
+      <Header />
+      <Switch>
+        <Route exact path="/" component={Home} />
+        <Route path="/about" component={About} />
+        <Route exact path="/restaurants" component={Restaurants} />
+        <Route path="/restaurants/:restaurantId" component={RestaurantContainer} />
```

Comment:
```suggestion
        <Route path="/restaurants/:id" component={RestaurantContainer} />
```

id 라고만해도 명확할 것 같은데 어떨까요? ☺️